constants/perpetuals.ts

```diff
@@ -0,0 +1,26 @@
+import assert from "assert";
+// hard coded perpetual info, until theres a better way to get this info
+export const perpetuals = [
+  {
+    // this is a perp kovan contract
+    name: "New Perpetual Contract Test",
+    symbol: "NEW-PERP-TEST",
+    // this is hardcoded to a mainnet uniswap eth usdc pair for now. But in the future we should be able
+    // to derive the market or markets where the synthetic is being traded...
```

Comment:
This will come in handy for the future to derive the market for a given pair: https://uniswap.org/docs/v2/SDK/pair/

Reply:
yea i didnt know what else to do here, but ive included a subgraph call to get amm information based on the token pair or the id of the amm.